_crc32c.c

```diff
@@ -49,29 +52,37 @@ PyObject* crc32c_crc32c(PyObject *self, PyObject *args) {
 #else
 	"s*"
 #endif
-	"|I:crc32";
+	"|Ii:crc32";
 
-	if (!PyArg_ParseTuple(args, format, &pbin, &crc) )
+	if (!PyArg_ParseTupleAndKeywords(args, kwargs, format, kwlist, &pbin, &crc, &release_gil) )
 		return NULL;
 
 	bin_data = pbin.buf;
-	crc ^= 0xffffffff;
-	result = crc_fn(crc, bin_data, pbin.len);
-	result ^= 0xffffffff;
+	if ((release_gil < 0 && pbin.len >= 32768) || release_gil >= 1) {
+		Py_BEGIN_ALLOW_THREADS
+		crc ^= 0xffffffff;
+		result = crc_fn(crc, bin_data, pbin.len);
+		result ^= 0xffffffff;
+		Py_END_ALLOW_THREADS
+	} else {
+		crc ^= 0xffffffff;
+		result = crc_fn(crc, bin_data, pbin.len);
+		result ^= 0xffffffff;
+	}
 
 	PyBuffer_Release(&pbin);
 	return PyLong_FromUnsignedLong(result);
 }
 
 static
-PyObject *crc32c_crc32(PyObject *self, PyObject *args)
+PyObject *crc32c_crc32(PyObject *self, PyObject *args, PyObject *kwargs)
 {
 	if (PyErr_WarnEx(PyExc_DeprecationWarning,
-	                 "crc32c.crc32 will be eventually removed, use crc32c.crc32c instead",
-	                 1) == -1) {
+					 "crc32c.crc32 will be eventually removed, use crc32c.crc32c instead",
+					 1) == -1) {
```

Comment:
These are two lines which *partly* were indented by spaces, ontop of the tab-based indentation that seems to be the case for every other line. I cleaned that up to have roughly the same indentation, but with tabs.

If you really don't want *any* formatting changes in this MR, please let me know. I figured fine since it's just two lines and the only place in this file where indentation is a bit messy, it's fine to do it right now as long as it is in a separate commit.

Reply:
Yes, I'd rather not include any formatting changes if possible. I used to be strongly minded about tabs/spaces (indent with tabs, align with spaces), but with time I've accepted the fact that most people/editors/IDEs simply work better with only spaces. Still, I'd rather keep formatting differences away. Thanks!

Reply:
Removed the formatting. Never thought I'd have a discussion about tabs vs. spaces in the 21st century :smile: